docs/src/app/(public)/(content)/react/components/select/demos/object-values/css-modules/index.module.css

```
@@ -12,11 +12,11 @@
   outline: 0;
   border: 1px solid var(--color-gray-200);
   border-radius: 0.375rem;
+  background-color: white;
```

```suggestion
  background-color: canvas;
```
(and update all other ones)